lineage/states/stateCommon.py

```diff
@@ -41,67 +44,42 @@ def bern_estimator(bern_obs: np.ndarray, gammas: np.ndarray):
     return numerator / denominator
 
 
+@njit
 def gamma_LL(
-    logX: np.ndarray,
-    gamma_obs: list[np.ndarray],
-    time_cen: list[np.ndarray],
-    gammas: list[np.ndarray],
+    logX: npt.NDArray[np.float64],
+    gamma_obs: List[npt.NDArray[np.float64]],
+    time_cen: List[npt.NDArray[np.float64]],
+    gammas: List[npt.NDArray[np.float64]],
 ):
     """Log-likelihood for the optionally censored Gamma distribution.
     The logX is the log transform of the parameters, in case of atonce estimation, it is [shape, scale1, scale2, scale3, scale4]."""
     x = np.exp(logX)
     glnA = gammaln(x[0])
-    outt = np.zeros(len(x) - 1)
+    outt = 0.0
```

Comment:
I tried running the figures, but I get this error:
`
Untyped global name 'gammaln': Cannot determine Numba type of <class 'numpy.ufunc'>
File "lineage/states/stateCommon.py", line 57:
def gamma_LL(
    <source elided>
    x = np.exp(logX)
    glnA = gammaln(x[0])
    ^
`
I think it is because njit doesn't work with `scipy.special`..

Reply:
Run `poetry update` to install `scipy-numba`

Reply:
Thank you, that worked.